styles.css

```diff
@@ -1 +1,64 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container {
+    padding: 16px;
+    background-color: #fdefdc;
+    display: grid;
+    grid-row-gap: 16px;
+    grid-column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px;
+    grid-template-areas:
+            "a"
+            "b"
+            "c"
+            "d"
+            "e"
+            "."
+            "f";
+}
+
+@media screen and (min-width: 576px) {
+    .container {
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 50px 190px auto 100px 50px;
```

Comment:
Jouw vak b is nu steeds 256px hoog, vandaar dat ik 16 van de 190 heb afgetrokken. Verder moet auto en 100px worden omgewisseld want nu is vak d dynamisch...

Reply:
auto en 100 omgedraaid, goede, verder wel 190 aangehouden omdat dit in de uitwerkingen staat